pkg/k8s/k8s.go

```diff
@@ -354,3 +354,43 @@ func AssertPrunerCronjobWithContainer(c *clients.Clients, namespace, num string)
 		testsuit.T.Errorf("Cronjob with prefix tekton-resource-pruner not found in %v namespace", namespace)
 	}
 }
+
+func AssertCronjobPresent(c *clients.Clients, cronJobName, namespace string) {
+	err := wait.Poll(config.APIRetry, config.ResourceTimeout, func() (bool, error) {
+		log.Printf("Verifying if cronjob with prefix %v is present in namespace %v", cronJobName, namespace)
+		cjs, err := c.KubeClient.Kube.BatchV1().CronJobs(namespace).List(c.Ctx, metav1.ListOptions{})
+		if err != nil {
+			return false, err
+		}
+		for _, cj := range cjs.Items {
+			if strings.Contains(cj.Name, cronJobName) {
+				return true, nil
+			}
+		}
+		return false, nil
+	})
+	if err != nil {
+		assert.FailOnError(fmt.Errorf("Expected: cronjob with prefix %v to be present in namespace %v, Actual:cronjob with prefix %v to be not present in namespace %v", cronJobName, namespace, cronJobName, namespace))
```

Comment:
in both functions
* space after `Actual:`
* in different functions you are using `%v present` and `%v not present` which I like more